api/envoy/config/wasm/v2alpha/wasm.proto

```diff
@@ -0,0 +1,78 @@
+syntax = "proto3";
+
+package envoy.config.wasm.v2alpha;
+
+import "envoy/api/v2/core/base.proto";
+
+import "google/protobuf/any.proto";
+
+import "udpa/annotations/migrate.proto";
+import "validate/validate.proto";
+
+option java_package = "io.envoyproxy.envoy.config.wasm.v2alpha";
+option java_outer_classname = "WasmProto";
+option java_multiple_files = true;
+option (udpa.annotations.file_migrate).move_to_package = "envoy.extensions.wasm.v3alpha";
```

Comment:
This should now be v3 since we have frozen v3.

Reply:
Could you update https://github.com/envoyproxy/envoy/blob/master/api/STYLE.md#adding-an-extension-configuration-to-the-api ?

V3 is generated from v2 according to the docs, so I don't know where to start here.

Reply:
@htuch we still can have alpha things at v3, right? So v3alpha should be OK for WASM for now?

Reply:
TBH, not sure how this works, as this is the first API. We need to figure out if the tooling can support it once we cut 1.13.0. Of course, we _should_ support this, but as it stands, we only support a flow where we start from v2 right now.

Reply:
@jplevyak that was updated yesterday to reflect the new flow. Sorry about the confusion, most of the turbulence should be over.

Reply:
Are there any action items for me on this or are we good ?

Reply:
Please just change all references to `v3alpha` to `v3` and run `fix_format`.